features/support/electron.js

```diff
@@ -130,6 +135,7 @@ AfterAll(async function() {
     await printMainProcessLogs();
   }
   if (process.env.KEEP_APP_AFTER_TESTS === 'true') {
+    // eslint-disable-next-line
```

Comment:
@DeeJayElly we are missing details here too.

Reply:
Done